### PR TITLE
Chapter 01 (Terminology fixes)

### PR DESCRIPTION
**Pull request by pgoyal01**
Made changes to some of the terms.
Still need to be resolved:
1. Do Instance Types include sizing (number of CPUs, RAM, etc.) information or are they just the Basic, Network-Intensive, etc. Types?  
2. Still need to fix the SW/HW Profile and Configuration terms.

### CODE REVIEW

**Comment by demels** [comment]
I suggest to remove the abbreviation vAPP and use the definition for Virtual Application as defined in ETSI GS NFV 003:

Virtual Application (VA): more general term for a piece of software which can be loaded into a Virtual Machine
NOTE: A VNF is one type of VA.

**Comment by rabiabdel** [comment]
@pgoyal01 i think you have painstakingly replaced the principles section in this pull request with the original text, i have modified it.

**Comment by rabiabdel** [comment]
There is an issue related to this: https://github.com/cntt-n/CNTT/issues/58

File: doc/ref_model/chapters/chapter01.md [hunk 1]
```diff
@@ -53,23 +53,23 @@ This section defines the main terms used in this document; these deinitions are
...
 - **Virtual Machine (VM)**: virtualised computation environment that behaves like a physical computer/server. 
-  >_*Note:*_ a **VM** consists of all of the components (processor (CPU), memory, storage, interfaces/ports, etc.) of a physical computer/server.
-- **VM Instance**: an abstract describtion of a VM that implies it's capabilities and resource usage. It has two parameters:
-  - **instance type**: The type of NFVI of which the VM is expected to run on. It directly maps to the combination of the underlying NFVI SW Profile and NFVI HW Profile.
-  - **compute flavour**: defines the compute, memory, and storage capacity of a virtual compute resource. 
+  >_*Note:*_ a **VM** consists of all of the components (processor (CPU), memory, storage, interfaces/ports, etc.) of a physical computer/server. It is created using Instance Type together with sizing information or Compute Flavour. 
+- **VM Instance**: is a VM
+- **Instance type**: defines the capabilities of the physical compute server that the virtual compute resource can run on.
```
Comment (rabiabdel) [inline doc/ref_model/chapters/chapter01.md]: Instance Type and Compute flavour are not of "physical compute server"

**Comment by hdamker** [comment]
@pgoyal01 I have some further change proposals, please integrate them into your pull request if you agree:

- Delete the abbreviation "vApp" completely, it is vendor specific. If you search for vApp you will find almost only pages from or about VMWare (try "vAPP NFV" or "vApp OpenStack")
- VNFs are built from VNF Components, VNFC.e.g. VMs => VNFs are built from VNF Components (VNFC), a VNFC single instance is a 1:1 mapping to e.g. a VM or Container
- NFVi => NFVI
- Workload: Workload refers to software running on top of NFVI resources such as VMs or Containers => Workload: another general term referring to software running on (virtualized) compute resources. Most relevant workload categories in context of NFVI are: ...
   - Background: i see this term as the most general one for the software running on NFVI (or other virtualization/cloud infrastructure. The listed subcategories under workload are only important examples, but not exhaustive
- the definition of "Instance type" is not correct from my perspective. Previous one was better (see also comment from @rabiabdel)
- Tenant (Internal) Networks (a.k.a. vApp Network): delete "(a.k.a. vApp Network)", see above.
- Compute Node: abstract definition of a physical or virtual server: delete "or virtual server". I have never seen the term used for a virtual server, it is always referring to the hardware node from my perspective

**Comment by ASawwaf** [comment]
@hdamker 
I totally agree on most of your modification, Just small comment on:

- VNFs are built from VNF Components, VNFC.e.g. VMs => VNFs are built from VNF Components (VNFC), a VNFC single instance is a 1:1 mapping to e.g. a VM or Container

Sawaf : VNFs are built from one or more VNF component , and in most cases, the VNFC is a single VM or container 

Sawaf : Workload: it is a software ( VNF or APP ) that running over NFVI and consume the NFVI resources

**Comment by hdamker** [comment]
@ASawwaf: +1 for the simplified version for VNFs, I like it.
Also for Workload ... just not "APP" in it ... instead "(VNF or virtualized application)"

**Comment by ASawwaf** [comment]
agreed

**Comment by pgoyal01** [comment]
@rabiabdel, @hdamker and @ASawwaf 

Thanks all. Will make some of the suggested changes.

Original Definition Instance Type: "The type of NFVI of which the VM is expected to run on. It directly maps to the combination of the underlying NFVI SW Profile and NFVI HW Profile." Issues: what does "type of NFVI" mean -- since it refers to VM is expected to run on it implies the physical hardware? what is the combination of NFVI SW Profile and NFVI HW Profile? Maybe my poor wording of the sentence: "defines the capabilities of the physical compute server that the virtual compute resource can run on." ----> capabilities of the virtual compute resource and these capabilities must match the configuration (capabilities) of the physical compute server on which, say, the VM is going to be running on. And as I stated in my comments there are issues with the Profile definitions also.

VNFC -- I like Ahmed's "in most cases."

Compute node will use the ETSI 1.4.1 (2018-10).
WIll delete vApps -- will need to be deleted from other Chapters.

I will make the changes except for Instance Type until we can get a clearer definition.

**Comment by ASawwaf** [comment]
@pgoyal01 , my suggestion to use Openstack definition ( with small modification ) which is " A running VM, or a VM in a known state such as suspended, that can be used like a hardware server "

my defination: It is running VM , or VM in a Known state such as (run, stop , suspend ) that can be used by any VNF or Virtual application with well-defined flavour which acting as dedicated hardware server

**Comment by ASawwaf** [comment]
the above is for Instance 

for instance type: 
Describes the parameters of the various virtual machine images that are available to users; includes parameters such as CPU, storage, and memory. Alternative term for flavor.

**Comment by karinesevilla** [comment]
@pgoyal01, @rabiabdel 
Agree to move the definitions of instance type and compute flavours a level above.
Instance type must be independent of VM, we should replace VM by VNFC -> the type of NFVI on which VNFC is expected to run on.

File: doc/ref_model/chapters/chapter01.md [hunk 2]
```diff
@@ -53,28 +53,28 @@ This section defines the main terms used in this document; these deinitions are
...
```
Comment (rabiabdel) [inline doc/ref_model/chapters/chapter01.md]: Hi @pgoyal01 can we please replace the sentence "physical compute server" in both instance type and compute flavour with "NFVI". Thanks

**Comment by rabiabdel** [comment]
Hi @pgoyal01 can we please replace the sentence "physical compute server" in both instance type and compute flavour with "NFVI". Thanks

**Comment by ASawwaf** [comment]
@rabiaabdel, Just maybe it used interchangeably but OpenStack guys :) , also this is from openstack.org definition

**Comment by rabiabdel** [comment]
Hi @pgoyal01: i an not sure the relevance of an image here. 

can you please revert the terminology for instance type and instance to the **original** definition (as it seems it is the one we agree on here). it will be great if we can get this approved tomorrow.

**Comment by markshostak** [comment]
Approved 9/7/2019 as written

**Pull request merged**